content/docs/sidebar.json

```diff
@@ -635,10 +635,10 @@
                 "slug": "what-is-a-model-registry"
               },
               "add-a-model",
-              "view-models",
-              "download-models",
               "register-version",
-              "assign-stage"
+              "assign-stage",
+              "view-and-compare-models",
+              "download-models"
```

Comment:
Let's either move this or at least relabel it to "Use Models" (or maybe "Download and Deploy Models"?)